src/test/java/api/requests/ItemsInTransitReportTests.java

```diff
@@ -49,6 +53,16 @@
   private static final String DISCOVERY_DISPLAY_NAME = "discoveryDisplayName";
   private static final String PICKUP_LOCATION = "pickupLocation";
   private static final String REQUEST = "request";
+  private static final String CALL_NUMBER = "callNumber";
```

Comment:
Refer to `ItemProperties` class.

Reply:
This field is used just for testing purpose. There isn't such field in ItemProperties class for now.

Reply:
What do you mean by it is for testing purposes? Does that mean, the constant is only used in the tests?

Reply:
the constant is only used in the tests